tests/foreman/ui/test_oscappolicy.py

```diff
@@ -53,7 +53,7 @@ def test_positive_create_with_policy_name(self):
                         content=content_name,
                         name=policy_name,
                         period=OSCAP_PERIOD['weekly'],
-                        profile=OSCAP_PROFILE['rhccp'],
+                        profile=OSCAP_PROFILE['c2s_rhel6'],
```

Comment:
Basically, values that has brackets doesn't work for all selenium and FF versions (https://github.com/seleniumhq/selenium/issues/674). If that is really matter to choose these specific values - just let me know and I will do a rollback as I am pretty sure that it should work on CI, but not on my env and some others too.